conf/general.yml-example

```diff
@@ -223,3 +223,7 @@ LOGIN_REQUIRED: 0
 # If you want to stop new users from registering, set this to 1.
 # NB: This also disables all Facebook/Twitter logins.
 SIGNUPS_DISABLED: 0
+
+# Memcached host
+# This can be safely left out and will default to '127.0.0.1' even if not present.
+FMS_MEMCACHED_HOST: '127.0.0.1'
```

Comment:
The `FMS_` prefix is a side-effect of our internal DB variable setup - it's not needed for any other variables (it's all FMS :) ). So I'd go with `MEMCACHED_HOST`.

Reply:
Also, I'd perhaps put it next to the SMTP/Gaze config - it sort of fits in there in terms of placement of a perhaps-external service.